src/vl-map.js

```diff
@@ -48,10 +54,23 @@ export class VlMap extends vlElement(HTMLElement) {
         <slot></slot>
       </div>
     `);
-
+    this.__extent = [9928, 66928, 272072, 329072];
     this.__prepareReadyPromises();
   }
 
+  _extentChangedCallback(oldValue, newValue) {
+    if (newValue) {
```

Comment:
Is er geen gedrag gewenst wanneer extent verwijderd wordt?

Reply:
Done